content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,111 @@
+You'll need to [install](https://metallb.universe.tf/installation/) it in order to continue with this guide.
+
+Now you should have a working environment to start experimenting with Gateway-API. There are many great guides you can follow at https://gateway-api.sigs.k8s.io/guides/.
+
+### Migrating from Ingress to Gateway API using Ingress2Gateway
+Migrating from Ingress to the Gateway API may seem intimidating, but luckily Kubernetes SIG-Network provides a tool to simplify the process. Ingress2Gateway assists in the migration by converting your existing Ingress resources into Gateway API resources. To get started with ingress2gateway, you need to first install the tool.
+
+```
+go install github.com/kubernetes-sigs/ingress2gateway@latest
+```
+This will install ingress2gateway to $(go env GOPATH)/bin/ingress2gateway.
+
+Alternatively, you can build it from source using https://github.com/kubernetes-sigs/ingress2gateway#build-from-source.
+
+Once the tool is installed, you can use it to convert the ingress resources in your cluster to GatewayAPI resources.
+
+```
+ingress2gateway print
+```
+
+This above command will:
+
+1. Read your Kube config file to extract the cluster credentials and the current active namespace.
+1. Search for ingresses and provider-specific resources in that namespace.
+1. Convert them to Gateway-API resources (Currently only Gateways and HTTPRoutes).
+For other options refer run the tool with `-h` or refer to https://github.com/kubernetes-sigs/ingress2gateway#options
+
+Review the converted Gateway API resources, validate them, and then apply them to your cluster.
+
+## Wrapping up
+Achieving reliable, scalable and extensible networking has always been a challenging objective.
+The Gateway API is designed to improve the current Kubernetes networking standards like ingress and/or implementation specific annotations and CRDs.
+By focusing on modular and expressive API resources, and a role-oriented approach, the Gateway API makes it possible to describe a wider array of routing configurations and models while improving security and ensuring smooth collaboration across different teams (infrastructure and application teams).
+
+It is a Kubernetes standard API, consistent across different platforms and implementations and most importantly it is future proof. The Gateway API is considered the Ingress’ successor, it has a sub team under SIG-Network that actively work on it and manage the ecosystem and it is likely to receive more updates and community support.
+
+Ingress2gateway helps to ease the migration process for you, converting your existing Ingress resources to Gateway API resources. To make migrations _even easier_ Ingress2gateway has recently added extensions to enable converting from provider-specific solutions, enabling it to translate implementations specific CRDs or Ingress annotations to the relevant Gateway API configurations. 
```

Comment:
I'd write:
```suggestion
The `ingress2gateway` tool helps to ease the migration process for you, converting an existing Ingress object to equivalent objects from Gateway . To make migrations even easier, `ingress2gateway` has recently added extensions to enable converting from provider-specific solutions, enabling it to translate some common implementation-specific custom resources or Ingress annotations to the closest Gateway API equivalent.
```

Reply:
thanks, I have changed it to more or less what you suggested